README.md

```diff
@@ -69,5 +69,11 @@ Licensed under the Apache License, Version 2.0 (the "License");you may not use t
 
 [http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0)
 
+
+Icons — CC BY 4.0 License
+
+In the Font Awesome Free download, the CC BY 4.0 license applies to all icons packaged as
+.svg and .js files types.
+
```

Comment:
The Font Awesome used in this project is licensed under a [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/legalcode)